vllm/sequence.py

```diff
@@ -468,8 +474,8 @@ def lora_int_id(self) -> int:
 
     def get_last_latency(self, now: float) -> Optional[float]:
         """Sets the last token time for Request level timings."""
-        # If still in prefill phase, raise Error.
-        if self.is_prefill():
+        # If still in initial prefill phase, raise Error.
```

Comment:
what's the case it is not "initial"?

Reply:
This happens for fast-forward tokens - if there is more than one, we switch to prefill mode.